x-pack/filebeat/module/envoyproxy/README.md

```diff
@@ -0,0 +1,128 @@
+# Envoyproxy Module
+
+This is a filebeat module for envoy proxy. 
+
+## Caveats
+
+* Module is to be considered _beta_.
+
+## Download and install Filebeat
+
+Grab the filebeat binary from elastic.co, and install it by following the instructions.
+
+## Deployment Scenario #1: envoy native deployment
+
+Make sure to update envoy configuration to enable log plugin. This module assumes that envoy log
+entries will be written to /var/log/envoy.log. Should it be not the case, please point the module 
+log path to the path of the log file. 
+
+Update filebeat.yml to point to Elasticsearch and Kibana. 
+Setup Filebeat.
+```
+./filebeat setup --modules envoyproxy -e
+```
+
+Enable the Filebeat envoyproxy module
+```
+./filebeat modules enable envoyproxy
+```
+
+Start Filebeat
+```
+./filebeat -e
+```
+
+Now, the Envoy logs and dashboard should appear in Kibana.
+
+
+## Deployment Scenario #2: envoy for kubernetes 
+
+For Kubernetes deployment, the filebeat daemon-set yaml file needs to be deployed to the 
+Kubernetes cluster. Sample configuration files is provided under the `beats/deploy/filebeat` 
```

Comment:
I assume this is the Github repo path.
Since this will be probably used by admins who download a tarball or install a package, can you include a link to the https://github.com/elastic/beats/tree/master/deploy/kubernetes/filebeat folder?

Reply:
sure